cartographer_ros/cartographer_ros/pbstream_map_publisher_main.cc

```diff
@@ -73,15 +73,22 @@ void Run(const std::string& pbstream_filename, const std::string& map_topic,
   }
   CHECK(reader.eof());
 
-  ::ros::NodeHandle node_handle("");
-  ::ros::Publisher pub = node_handle.advertise<nav_msgs::OccupancyGrid>(
-      map_topic, kLatestOnlyPublisherQueueSize, true /*latched */);
-
   LOG(INFO) << "Generating combined map image from submap slices.";
   const auto painted_slices =
       ::cartographer::io::PaintSubmapSlices(submap_slices, resolution);
   std::unique_ptr<nav_msgs::OccupancyGrid> msg_ptr = CreateOccupancyGridMsg(
       painted_slices, resolution, FLAGS_map_frame_id, ros::Time::now());
+  return msg_ptr;
```

Comment:
don't give it a name, just return Create...?

Reply:
:+1: